### PR TITLE
[JENKINS-62278] Jenkins.MANAGE access global config

### PR DESCRIPTION
**Pull request by amuniz**
[JENKINS-62278](https://issues.jenkins-ci.org/browse/JENKINS-62278)

Users with `Jenkins.MANAGE` should be able to configure global settings.
(`Jenkins.MANAGE` requires a core bump, and I'm also updating the parent).

### CODE REVIEW

**Comment by amuniz** [comment]
@jglick I see you are merging PRs and doing releases here recently. Could you have a look to this please?

File: src/main/java/hudson/scm/SubversionSCM.java [hunk 1]
```diff
@@ -2197,6 +2198,12 @@ public boolean isBrowserReusable(SubversionSCM x, SubversionSCM y) {
             return true;
         }
 
+        @Nonnull
+        @Override
+        public Permission getRequiredGlobalConfigPagePermission() {
+            return Jenkins.MANAGE;
```
Comment (EstherAF) [inline src/main/java/hudson/scm/SubversionSCM.java:2204]: Changing this permission check is enough? there is no other permission check in the submitting, or in the jelly template?

Comment (EstherAF) [inline src/main/java/hudson/scm/SubversionSCM.java:2204]: Changing this permission check is enough? there is no other permission check in the submitting, or in the jelly template?

Comment (amuniz) [reply src/main/java/hudson/scm/SubversionSCM.java:2204]: > there is no other permission check in the submitting

This is submitted as part of the global configuration and it is already [checking for Manage](https://github.com/jenkinsci/jenkins/blob/78ea4b6b558362356f9a59b627fdc78ad21e4569/core/src/main/java/jenkins/model/Jenkins.java#L3803)

> or in the jelly template?

No, this is a `global.jelly`, already handled by core.

**Comment by jglick** [comment]
> @jglick I see you are merging PRs and doing releases here recently.

Am I? Not by design. Would like to see someone officially pick up at least basic maintenance of this.

```yaml
developers:
- "jglick"
- "kutzi"
- "recena"
- "schristou"
- "stephenconnolly"
- "oleg_nenashev"
- "ifernandezcalvo"
```

**Comment by amuniz** [comment]
Maybe @oleg-nenashev or @stephenc?

**Comment by amuniz** [comment]
> Would like to see someone officially pick up at least basic maintenance of this

Me too.

**Comment by oleg-nenashev** [comment]
Please feel free to step up as a maintainer! Contributions are welcome!

**Comment by amuniz** [comment]
> Please feel free to step up as a maintainer! Contributions are welcome!

Sorry, don't have the time for it.

**Comment by mikecirioli** [comment]
can we at least merge this change, since there is no owner to object?

**Comment by oleg-nenashev** [comment]
This not how the merge process works. Merge permissions are a part of the
plugin maintenance. There is no value in merging just to create more
obstacles and unreleased technical debt for a new maintainer

**Comment by jtnord** [comment]
there are a few validation methods that still check for ADMINISTER - should these be updated or are they expected to remain as ADMINSTER as they are credential related?

**Comment by amuniz** [comment]
@jtnord yes, those should remain as ADMINISTER.
Conflicts fixed.

@timja I see you created some releases of this plugin recently, could you have a look to this please?

File: pom.xml [hunk 2]
```diff
@@ -255,6 +257,12 @@ THE SOFTWARE.
       <artifactId>workflow-cps</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- Extracted from core at 2.190.1 -->
+    <dependency>
```
Comment (timja) [inline pom.xml:261]: not needed, ssh-credentials pulls it

**Comment by timja** [comment]
> @timja I see you created some releases of this plugin recently, could you have a look to this please?

To be clear, I have no access to svn or any intention to do anything to this plugin other than merging incredibly simple PRs or breaking changes in core.

This plugin is up for adoption.

(I can merge and release this PR though)

File: pom.xml (hunk 2 above)
Comment (amuniz) [reply pom.xml]: :+1:

**Pull request merged**